sycl/test-e2e/Matrix/SPVCooperativeMatrix/SG32/joint_matrix_bf16_fill_k_cache_unroll_init.cpp

```diff
@@ -0,0 +1,24 @@
+//==--joint_matrix_bf16_fill_k_cache_unroll_init.cpp  - DPC++ joint_matrix--==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// SG size = 32 is not currently supported for SYCL Joint Matrix by IGC on DG2
+// UNSUPPORTED: gpu-intel-dg2
+// REQUIRES: gpu, aspect-ext_intel_matrix, gpu
+// REQUIRES-INTEL-DRIVER: lin: 27501, win: 101.4943
+
+// RUN: %{build} -mllvm -inline-threshold=5000 -ffp-model=precise -o %t_gpu.out -DINIT_LIST -DMANUAL_UNROLL
```

Comment:
@MrSidims , it seems you need to correct your script or do a manual review of the changes.
I'm stopping review here.